package.json

```
@@ -8,6 +8,7 @@
     "test": "web-test-runner --node-resolve --coverage",
     "test-ci": "web-test-runner --node-resolve --coverage",
     "test:watch": "web-test-runner --node-resolve --watch --coverage",
+    "build-standalone": "rollup --config",
```

```suggestion
    "build-standalone": "rollup --config",
    "prepack": "build-standalone",
```

I think this does the trick.